public/js/components/Editor.js

```diff
@@ -221,22 +223,30 @@ const Editor = React.createClass({
     }
   },
 
-  render() {
+  renderBreakpoints() {
     const { breakpoints, sourceText } = this.props;
     const isLoading = sourceText && sourceText.get("loading");
 
+    if (isLoading) {
+      return;
+    }
+
+    return breakpoints.valueSeq().map(bp => {
+      return Breakpoint({
+        key: makeLocationId(bp.location),
+        breakpoint: bp,
+        editor: this.editor && this.editor.codeMirror
+      });
+    });
+  },
+
+  render() {
     return (
       dom.div(
         { className: "editor-wrapper devtools-monospace" },
         dom.div({ className: "editor-mount" }),
-        !isLoading &&
-          breakpoints.valueSeq().map(bp => {
-            return Breakpoint({
-              key: makeLocationId(bp.location),
-              breakpoint: bp,
-              editor: this.editor && this.editor.codeMirror
-            });
-          })
+        this.renderBreakpoints(),
```

Comment:
I'm ok keeping this as is, but generally we should talk about style so it doesn't keep coming up. I think it's more flexible to accept that things like ternaries are how you do conditional templating w/React, and trying to avoid it is going to break things up too much in my opinion. Once you're used to it, it's really nice to look at one place and read what it renders, instead of always splitting off into `render*` functions, and it's much easier to refactor. Of course, if it's any longer than ~10 lines or requires complicated logic, that's perfectly fine to do. It's subjective, but I wouldn't say we should enforce things like avoiding ternaries.

I personally don't mind this code, but one thing that would help clean it up for some people is to use JSX. I'm fine with that too, although I don't know how much work it would be to convert to it.